int32 camera_index
string camera_name
int32 width
int32 height
BlobMove[] data
